### DISCUSSION TITLE
tmdb_discover attribute not allowed with Collection Level: Episode

### DISCUSSION THREAD

**Discussion by snickers2k** [meisnate12/Plex-Meta-Manager#627] [Q&A]
is there any reason why tmdb_discover does not work with collection_level: episode ?

`Collection Error: tmdb_discover attribute not allowed with Collection Level: Episode`

or should i feature request that one?^^
thanks

**Comment by meisnate12** [comment]
because the [api endpoint](https://developers.themoviedb.org/3/discover/tv-discover) returns shows not episodes
